internal/db/migrations/atlas.sum

```diff
@@ -9,4 +9,5 @@ h1:4nZSKvUlpsK5RT4Ctn1jUnwvnDQRGaEYafWJxJ7s95w=
 20250402173826_remove_workflow_type.up.sql h1:P0w5tngLlCsfyYm1hW8LqdxDgVAYWIf9QM7oTLu5mJA=
 20250418000742_update_sip_statuses.up.sql h1:FUbEzeO8+t0FVImqnOWApQqKzxulq2GY4urMSsJlbc0=
 20250509030046_restore_sip_status_index.up.sql h1:8iKBW+nC9mgEs6PqQJ4Frq1IGJOjedAyhm3dO27yVZw=
-20250509030521_add_sip_uuid_column.up.sql h1:z5zEsQbzL/nz4q7MgBdkiQa+KKB5rPopYrte0lRrst4=
+20250509030521_add_sip_uuid_column.up.sql h1:rtcC2HuAOJZI+/7ZkWrlrTn+wkrZ34BRqLK5bJRo3h8=
```

Comment:
Looks like the previous hash was not right, I'll think about adding CI to check for missing migrations and things like this.